### PR TITLE
"News" section > "Press Releases" section

### PR DESCRIPTION
**Pull request by bfauble**
#### Summary of changes
**Asana Ticket:** ["News" section > "Press Releases" section](https://app.asana.com/0/555089885850811/1202075661172628/f)

image
image

### CODE REVIEW

File: apps/site/test/site_web/views/page_view_test.exs [hunk 1]
```diff
@@ -72,9 +72,9 @@ defmodule SiteWeb.PageViewTest do
         |> PageView.render_news_entries()
         |> HTML.safe_to_string()
 
-      assert rendered |> Floki.find(".c-news-entry") |> Enum.count() == 5
-      assert rendered |> Floki.find(".c-news-entry--large") |> Enum.count() == 2
-      assert rendered |> Floki.find(".c-news-entry--small") |> Enum.count() == 3
+      assert rendered |> Floki.find(".c-news-entry") |> Enum.count() == 6
+      # assert rendered |> Floki.find(".c-news-entry--large") |> Enum.count() == 2
```
Comment (i0sea) [inline apps/site/test/site_web/views/page_view_test.exs:76]: Should this be uncommented or deleted?

**Review by i0sea — COMMENTED** [review]
Looking at the mocks there's some padding on top of the icons.  xs screens may be limited to 3 press releases but I'm unsure if that's accurate. Otherwise LGTM

**Comment by lund5000** [comment]
> xs screens may be limited to 3 press releases

Good catch, @i0sea! We added that detail to the events task but missed getting it into this one. 🙈 

@bfauble If it's not a quick update we can defer adding it, but it would be good to limit the list to three items at the xs breakpoint (six at higher breakpoints).

**Review by thecristen — CHANGES_REQUESTED** [review]
I'm ok merging this if we remove the soon-to-be-extraneous "upcoming events" link.

**Pull request merged**